requirements/openvino.txt

```diff
@@ -2,9 +2,9 @@
 # OpenVINO Requirements.                                                      #
 # FIXME: nncf 2.3.0 has scipy version conflict
 # this is a temporary workaround to udpate nncf dependencies
-nncf @ git+https://github.com/cih9088/nncf@9b204857c11bc694c2ca6a62fc9a575c002a0a99#egg=nncf
+nncf==2.4.0
```

Comment:
@cih9088 You don't need to create a separate PR :)

Reply:
Thank you! Could you remove the comment above please? 😊